packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -843,6 +843,10 @@ class AnimationController extends Animation<double>
   }
 
   void _tick(Duration elapsed) {
+    if (_ticker == null) {
```

Comment:
This looks fishy. Why is `_tick` still being called when the associated ticker has been disposed?

Reply:
Because ticker uses `SchedulerBinding.instance.scheduleFrameCallback` to invoke tick.
There is number of cases of usage after disposal in Flutter, as it is validated only for small subset. It is not in scope of current cleanup to fix all these cases.
Current clean up makes sure to dispose all disposables, even when usage after disposal happens.

Reply:
Yeah, but it is unscheduled when the ticker is disposed. So, if _tick is still called after dispose there is something else that is not working as intended and - on the surface - this just seems to paper over that problem.

Reply:
I could not repro the problem I was trying to solve with this. Removed.